### ISSUE TITLE
paper: needs title

### ISSUE THREAD

**Issue by fsolt** [fsolt/dcpo_gender_roles#15]
We need titles for the dataset and for the paper.  

My thoughts re the dataset: `EGR` (as in "EGR data") *might* be [taken](https://www.egrdatabriefing.com/) from a [SEO standpoint](https://en.wikipedia.org/wiki/Search_engine_optimization), but `EGRD` ("Egalitarian Gender Roles Database") would probably work fine.  I'd kind of prefer "Egalitarian Gender Attitudes Database", but `EGAD` is obviously a nonstarter.  I'm not sure "Egalitarian Gender Opinion Database" (`EGOD`) works any better at all, really, and it definitely doesn't work for SEO.  Mixing "Roles" with "Attitudes" or "Opinion" gives `EGRAD` (nope--hits like every online graduate degree program) or `EGROD`, which I suppose is okay if not particularly mellifluous.  Clearly no obvious winner here, so let's brainstorm, please. 

My thoughts re the paper: "The [dataset title]: A Dataset of Dynamic Comparative Public Opinion Toward Egalitarian Gender Roles" might work fine?  More brainstorming needed.

**Comment by byngdeuk** [comment]
I like the title: "The [dataset title]: A Dataset of Dynamic Comparative Public Opinion Toward Egalitarian Gender Roles" !
I'm pretty bad at naming dataset.. But, I will do brainstorming!!

**Comment by fsolt** [comment]
Another alphabet-soup option (tagging @lallemang so she can start thinking about this in her free time too lol) is `GEA` "Gender Egalitarian Attitudes".  My SEO check (i.e., googling "GEA data") on that wasn't awful.  First hit for GEAD was Urban Dictionary, though.  Very bad.

**Comment by lallemang** [comment]
AEGR: "Attitudes toward Egalitarian Gender Roles"

When I Googled "AEGR dataset", the only data-related hit that came up on the first page was for an article on machine learning that, "proposes a new gradient-reversal method... to overcome the influence of anomalies on the training phase for the purpose of detecting network anomalies." ([see link](https://arxiv.org/abs/1912.13387)) A pharmaceutical company also pops up on the first Google page. The first hit on the second Google page is the "Annual Electric Generator Report" from data.gov. So maybe that's a concern for us, although it doesn't pop up until you're on the second page.

I'll keep brainstorming name ideas!

**Comment by sammo3182** [comment]
Just quick thoughts for your reference: A2G ("Attitude towards Gender"), GGI (Global Gender Ideology/Index), GEE (Gender Equality Evaluation. There's a Google Earth Engine, though, probably not a good one), or EofG (Equality of Gender).

**Comment by fsolt** [comment]
`EGA` for "Egalitarian Gender Attitudes"?  I hadn't even written it down before because googling "ega data" gives a bunch of hits for the European Genome-phenome Archive.  I still like it though, and the spirit of brainstorming says evaluate later.

**Comment by fsolt** [comment]
Oh, I remembered the name of the nerd-fest extracurricular I mentioned yesterday that I did in junior high that taught me to brainstorm: [Future Problem Solving](https://en.wikipedia.org/wiki/Future_Problem_Solving_Program_International).  It was super-fun.

**Comment by fsolt** [comment]
As we've tightened our focus to equality in the traditionally male-dominated public sphere, maybe we should indicate this somehow in the name (and title)?  The `PGE` Database (for Public Gender Egalitarianism)?  Not too much confusion, I don't think, with "[Pacific Gas & Electric](https://www.pge.com/)" or "[Portland General Electric](https://www.portlandgeneral.com)," the two top hits I got when searching PGE.

Or maybe we don't really need to worry about "public" at all, and we can go back to all the options we had listed back in the summer?  Thoughts?
